mitodl/bootcamp-ecommerce
# Remove old payment page

#### What are the relevant tickets?
Closes #849

#### What's this PR do?
Removes the payment page url and react code

#### How should this be manually tested?
- http://bc.odl.local:8099/pay should 404
- Use hijack from django admin to log in as another user.  You should end up on the `/applications` url (on master you'd end up on the /pay url)
